docs/module_guides/evaluating/evaluating_with_llamadatasets.md

```diff
@@ -0,0 +1,146 @@
+# Evaluating With `LlamaDataset`'s
+
+We have already gone through the core abstractions within the Evaluation module that
+enable various kinds of evaluation methodologies of LLM-based applications or systems, including RAG systems. Of course, to evaluate the system one needs an
+evaluation method, the system itself, as well as evaluation datasets. It is
+considered best practice to test the LLM application on several distinct datasets
+emanating from different sources and domains. Doing so, helps to ensure the overall
+robustness (that is, the level in which the system will work in unseen, new cases) of
+the system.
+
+To this end, we've included the `BaseLlamaDataset` (referred to as `LlamaDataset`
```

Comment:
Do we need to mention `BaseLlamaDataset` at all here? Seems like it could be removed and have the text focus on `LaballedRagDataset`

Reply:
Yeah, you're probably right here. I'll simplify and just refer to `LabelledRagDataset`. Thanks!